autoreduce_frontend/static/css/base.css

```diff
@@ -82,7 +86,7 @@ footer {
     text-align: center;
     margin-top: 30px;
     color: #555555;
-    margin-bottom: 10px;
+    margin-bottom: 250px;
```

Comment:
```suggestion
```
Doesn't look like we need this margin as I always see the scroll there now with `overflow-y`